docs/developer/process/agent-release.md

```diff
@@ -112,4 +118,4 @@ ddev agent changelog
 ddev agent integrations
 ```
 
-Create a pull request and wait for approval before merging.
+Update the [`AGENT_CHANGELOG`][agent-changelog] and [`AGENT_INTEGRATIONS`][agent-integrations] files with the outputs of these commands. Create a pull request and wait for approval before merging.
```

Comment:
Link commands

Reply:
The `ddev agent changelog/integrations` commands?

Reply:
updated!